index.html

```diff
@@ -1,312 +0,0 @@
-                                                width=40% height=40%>
-
-
-                                          <p>In this way, the student can be guided from the simple concept of a
-                                                single-degree-of-freedom
-                                                structural system to the more complex multidegree-of-freedom and
-                                                continuous
-                                                systems,
-                                                including random
-                                                vibrations, nonlinear systems, and aeroelastic phenomena.
-                                          </p>
-                                          <div class="linkButton">
-                                                <button id="pic-link-button"><a href="blogPost.html"
-                                                            target="_blank">Learn more!</a></button>
-                                          </div>
-                                    </div>
-                                    <div class="articleBox">
-                                          <h3>Plane flies Upside-down!</h3>
-                                          <img class="articlePicture" id="articlePic12" src="images/002.jpg" width=40%
-                                                height=40%>
-
-                                          <p>In this way, the student can be guided from the simple concept of a
-                                                single-degree-of-freedom
-                                                structural system to the more complex multidegree-of-freedom and
-                                                continuous
-                                                systems,
-                                                including random
-                                                vibrations, nonlinear systems, and aeroelastic phenomena.
-                                          </p>
-                                          <div class="linkButton">
-                                                <button id="pic-link-button"><a href="blogPost.html"
-                                                            target="_blank">Learn more!</a></button>
-                                          </div>
-                                    </div>
-
-                              </div>
-                        </div>
-                  </div>
-            </div>
-
-            <div class="footerContainer">
-                  <div class="footer">
-
-
-                        <div class="footerTitle">QUESTIONS OR COMMENTS? <SPAN id="footerTitleBold">GET IN TOUCH:</SPAN>
-
-                        </div>
-
-                        <div class="formSection">
-
-                              <form action="action_page.php">

```

Comment:
Here you don't need this action filled out since your form isn't actually supposed to be doing anything, but this is how you would send your information to a another file for further processing.